src/devices/Nrf24l01/Nrf24l01.cs

```diff
@@ -545,11 +562,13 @@ internal WorkingMode ReadWorkwingMode()
         /// <param name="power">Output Power</param>
         internal void SetOutputPower(OutputPower power)
         {
+            // Details in the Datasheet P54
+            // Register: 0x06, bit[2:1]
             _gpio.Write(_ce, PinValue.Low);
 
             Span<byte> readData = WriteRead(Command.NRF_R_REGISTER, Register.NRF_RF_SETUP, 1);
 
-            byte setting = (byte)(readData[0] & (~0x06) | ((byte)power << 1));
+            byte setting = (byte)(readData[0] & (~0b_0000_0110) | ((byte)power << 1));
```

Comment:
why not directly write negated bits?

Reply:
actually whichever since this can be thought of as "clear these bits"

Reply:
I just want to make the operation meaningful and readable.